Solutions/Menes.Hosting.AspNetCore/Menes/Internal/OpenApiHttpResponseResult.cs

```diff
@@ -315,7 +317,15 @@ private void BuildHeaders(HttpResponse httpResponse, OpenApiResponse response)
                         convertedValue = this.ConvertValue(header.Value.Schema, value);
                     }
 
-                    httpResponse.Headers.Add(header.Key, new Microsoft.Extensions.Primitives.StringValues(convertedValue));
+                    // If the input value was a string, it will have been returned as if it were a serialized JSON element.
+                    // This means it will be quoted, which we don't want for values going in the headers, so we'll get rid
+                    // of the quotes if they are present.
+                    if (convertedValue[0] == '"')
+                    {
+                        convertedValue = convertedValue[1..^2];
```

Comment:
I think that is going to lop TWO characters off the end.

If you're addressing individual characters, `convertedValue[^2]` would be the penultimate one, so that would get the the last character before the closing `"`. But when you slice with a range, the end index refers to the end of the slice that you want, and not the start of the final character within the slice that you'd like.

Here's the image in my book I use to show it:

![image](https://user-images.githubusercontent.com/334181/223121089-1f746374-2d0f-4c6c-9bd0-d41f22646ebe.png)

An `Index` doesn't identify an element, it identifies the boundary between two elements (or the very start/end of the input as appropriate).

`^1` points to the gap between the last-but-one and the final item. When addressing individual items, that will give you the last but one item because it points right to the start of that item. When describing the end of a range, that will exclude the last but one item because it points right to the start of that item.